packages/api/src/external/carequality/patient.ts

```diff
@@ -25,41 +29,45 @@ import {
   getOutboundPatientDiscoveryRespCount,
   getOutboundPatientDiscoveryResps,
 } from "./command/outbound-patient-discovery-resp/get-outbound-patient-discovery-resp";
-import { createPatientDiscoveryRequest } from "./create-pd-request";
 import { CQLink } from "./cq-patient-data";
-import { OutboundPatientDiscoveryResp } from "./patient-discovery-result";
+import { createPatientDiscoveryRequest } from "./create-pd-request";
 import { cqOrgsToXCPDGateways, generateIdsForGateways } from "./organization-conversion";
-import { MedicalDataSource } from "@metriport/core/external/index";
+import { OutboundPatientDiscoveryResp } from "./patient-discovery-result";
 import { PatientDataCarequality } from "./patient-shared";
-import { getCQGateways } from "./command/cq-directory/cq-gateways";
 
 dayjs.extend(duration);
 
+const context = "cq.patient.discover";
+const iheGateway = makeIheGatewayAPIForPatientDiscovery();
+
+export const PATIENT_DISCOVERY_TIMEOUT = dayjs.duration({ minutes: 0.25 });
+const CHECK_DB_INTERVAL = dayjs.duration({ seconds: 5 });
```

Comment:
couple of comments:

Why not just do `PATIENT_DISCOVERY_TIMEOUT = dayjs.duration({ seconds: 15 });` to be consistent with the below var, so the dev doesn't have to do mental math?

Some big picture questions:
- are we waiting 15s for PD? I thought we were moving away from long running ops on the API in case of restarts?
- I don't see any code that marks the PD as "completed", how do we know that we can do DQs after they're triggered? If the cx triggers a DQ right after a PD, it seems like we will proceed with the DQ even if there's a single link. Am I missing something, or is this an oversight in the design?

Reply:
Happy to review the code, but this was already there, I just moved it up to keep constants at the top of the file.

> `PATIENT_DISCOVERY_TIMEOUT = dayjs.duration({ seconds: 15 });`

💯 

> are we waiting 15s for PD? I thought we were moving away from long running ops on the API in case of restarts?

I don't remember right now, but maybe @RamilGaripov does. I agree it would be better to have a lambda like we do for DQ/DR, but that's likely a ticket at this point. 😅 

> I don't see any code that marks the PD as "completed"...

It's marked as completed when we create the entry on `cq_patient_data` w/ the CQ links. And that's only created when either we have responses from all gateways or it timed out.

That brings a good point about not DQ'ing if no entry exists on that table. Added it to the next commit.